javac-plugin/src/test/java/org/moditect/deptective/internal/graph/CyclesTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ *  Copyright 2019 The ModiTect authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.moditect.deptective.internal.graph;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.moditect.deptective.internal.graph.fwk.TestModelCreator;
+
+/**
+ * 
+ * @author Gerd W&uuml;therich (gw@code-kontor.io)
+ */
+public class CyclesTest {
+
+	@Test
+	public void detectCycle() {
+
+		//
+		List<INode> nodes = TestModelCreator.createDummyModel();
+
+		//
+		List<List<INode>> cycles = GraphUtils.detectCycles(nodes);
+
+		//
+		assertThat(cycles).hasSize(1);
+
+		//
```

Comment:
What is the meaning of these comments? I saw it at several places, is there something special behind it?